renderer/containers/preferences.js

```diff
@@ -86,6 +86,10 @@ export default class PreferencesContainer extends Container {
         pluginsFromNpm: pluginsFromNpm.filter(p => p.name !== name),
         pluginsInstalled: [plugin, ...pluginsInstalled].sort((a, b) => a.prettyName.localeCompare(b.prettyName))
       });
+    } else {
+      this.setState({
+        pluginBeingInstalled: null
```

Comment:
Can we try to use `undefined` whenever possible over `null`? See https://github.com/sindresorhus/meta/issues/7